code/game/objects/items/devices/radio/detpack.dm

```diff
@@ -321,5 +322,16 @@
 		processing_timers.Remove(src)
 	return
 
+/obj/item/device/radio/detpack/proc/follow_target(var/atom/movable/target)

```

Comment:
The way to handle this is with visual contents, really.

Reply:
It's already been tried.

It appears to be a prefab list.

Adding the detpack to it appears to do nothing; it is still completely obscured when it then is put into the target object; likewise for adding the object instead, or adding both.

Some help would actually be welcome, because there isn't any instance of its use whatsoever in the code, and google has turned up little.

Reply:
Sent you examples of usage in other codebases in discord.